### PR TITLE
pr- Olivia Charles 

### PR DESCRIPTION
**Pull request by ochar721**

### CODE REVIEW

File: lib/character.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Character
+  attr_reader :name,
+              :actor,
+              :salary
+
+  def initialize(name:, actor:, salary:)
```
Comment (robertguturing) [inline lib/character.rb:6]: The character comes in a single argument as a hash so this wouldn't work. You would need to do

```
def initialize(data)
  @name = data[:name]
  @actor = data[:actor]
  @salary = data[:salary]
end
```

File: lib/network.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class Network
+    attr_reader :name,
+                :shows
+
+  def initialize(network)
+    @network = network
+    @name    = "NBC"
```
Comment (robertguturing) [inline lib/network.rb:7]: I think you want the name here to be dynamic and passed in as an argument to the `initialize` method. That way we can have a network name other than "NBC"

**Review by robertguturing — COMMENTED** [review]
Score : 1

Keep on working on class creation and what arguments should be passed into initialize!

File: lib/character.rb (hunk 1 above)
Comment (ochar721) [reply lib/character.rb:6]: yep, i really struggled with this entire thing clearly. had a panic attack and froze. I went over everything with madeleine yesterday

**Pull request closed without merging**